docs/examples/ffmpeg-video-processing.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: "Video processing with FFmpeg"
+sidebarTitle: "FFmpeg video processing"
+description: "These examples show you how to process videos in various ways using FFmpeg with Trigger.dev."
+---
+
+## Adding the FFmpeg build extension
+
+To use these example tasks, you'll first need to add our FFmpeg extension to your project configuration like this:
```

Comment:
Should probably mention the need to add `@trigger.dev/build` to your `devDependencies` if it already isn't there (`init` will now add it).